libs/src/api/rewards.js

```diff
@@ -16,12 +16,10 @@ const GetAttestationError = Object.freeze({
   USER_NOT_FOUND: 'USER_NOT_FOUND',
   HCAPTCHA: 'HCAPTCHA',
   COGNITO_FLOW: 'COGNITO_FLOW',
-  BLOCKED: 'BLOCKED',
-  OTHER: 'OTHER',
   DISCOVERY_NODE_ATTESTATION_ERROR: 'DISCOVERY_NODE_ATTESTATION_ERROR',
   DISCOVERY_NODE_UNKNOWN_RESPONSE: 'DISCOVERY_NODE_UNKNOWN_RESPONSE',
   AAO_ATTESTATION_ERROR: 'AAO_ATTESTATION_ERROR',
-  AAO_ATTESTATION_UNKNOWN_RESPONSE: 'AAO_ATTESTATION_UNKNOWN_RESPONSE',
+  AAO_ATTESTATION_FAILURE: 'AAO_ATTESTATION_FAILURE',
```

Comment:
So if this error represents a generic rejection, maybe we should just call it AAO_ATTESTATION_REJECTED or something. AAO_ATTESTATION_FAILURE sounds like something went wrong on the AAO side, which this doesn't indicate